### ISSUE TITLE
[Project] Himalayan Expeditions

### ISSUE THREAD

**Issue by jessimckenzi** [jsoma/data-studio-projects#202]
# Pitch

I'm curious how different variables impact success rates in climbing Himalayan mountains, including the use of oxygen, and whether or not the expedition team included women.

## Summary

The Himalayan Database includes data on each Himalayan peak, known expeditions, and known expedition team members—there's a lot there! I'm struggling to figure out how to assess success rates for subgroups within the dataset.

## Details

**Possible headline(s):** 
The Making of a Successful Summit Attempt

**Data set(s):** 
http://www.himalayandatabase.com/index.html

**Code repository:** 
https://github.com/jessimckenzi/data-studio/tree/master/03-expeditions/Himalayan%20Database/HIMDATA
	
**Possible problems/fears/questions:** 
Probably because it's late, but there are lots of questions I have that I'm struggling to figure out how to approach. These are some questions I'd like to ask this dataset:

Do expedition groups with women have a higher or lower frequency of deaths? Do expedition groups with women have a higher or lower success rate compared to all-men groups? What is the greatest lapse of time between first (known) attempt and first successful summit? Does the use of oxygen increase the likelihood of success or death? Which peak has the most successful rate of summitting? The least? Which is the most deadly peak?

I feel like a lot of these questions require the same technique: creating a subgroup and then analyzing that, but I'm just struggling to make it happen right now.

## Work so far
Ok I don't have images of my own to share on this one yet, but here's inspiration, from someone else who has looked at similar (or the same) data
![screen shot 2018-07-23 at 10 51 57 pm](https://user-images.githubusercontent.com/39416177/43114310-721109f6-8ecc-11e8-90cd-bdeabab48131.png)
![screen shot 2018-07-23 at 10 51 29 pm](https://user-images.githubusercontent.com/39416177/43114311-7221c0d4-8ecc-11e8-9ff5-932def8776e1.png)

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by Palarisk** [comment]
This is a really interesting project. You have so many good questions so you just have to pick where to concentrate. I'd definitely want to know if there's huge differencies in fatalities between groups with women and all-men groups. Just keep in mind not to claim that the differences would be caused because of women/no-women, especially because the overall numbers here are so small, one death can have a huge impact on how the numbers look like.

**Comment by jessimckenzi** [comment]
# Update
I made a terrible mistake. Instead of pursuing some of the great research questions I posed in my original post, I decided I wanted to know what the most common types of fatal accidents are in the Himalayas. "It's just a little regular expression cleaning," I told myself. No. Each entry is entered almost exactly as someone described it, with no consistency. So I had to search for all sorts of different work combinations, like "fatal fall" "killed in fall" "fell, body recovered" "fell, body not found" etc. SO MANY VARIATIONS. And it took forever to eliminate illness, or injuries, because sometimes it would be "broken leg; fatal fall" or similar. I should have seen this was a huge problem and called it quits, but didn't.

## Your project content: images/words/etc

The most common fatal accidents during climbing expeditions in the Himalayas

![screen shot 2018-07-29 at 10 07 12 pm](https://user-images.githubusercontent.com/39416177/43374060-c328f2aa-937b-11e8-970e-0e8c93c13491.png)

  
## Any changes in direction or topic?
I completely ignored what I said I was going to do, but I actually am still really interested in it and want to do it this week.

	
## Problems/Questions

My carefully and laboriously crafted waffle chart looks like an optical illusion.

I am still a bit lost as to go about calculating success rates when supplemental oxygen is used or not. My brain is just struggling to compute!
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
nice start. love that you show deaths and then the rate.

so your waffle chart is buzzing because you need to use more color to distinguish between the categories. the grays are too close together for us to really sort out the data. you could also attach the labels to the chart, just put them on the chart. once you have the color you just put the label next to the color and we don't have to work so hard to sort it out. i'd also make falling rock and falling ice the same color family, because they are both 'something falls and kills you' so light blue, dark blue perhaps. they relate, so show us that with color.

**Comment by jessimckenzi** [comment]
# Final

## Project visuals/text
![screen shot 2018-08-16 at 10 44 18 pm](https://user-images.githubusercontent.com/39416177/44245295-0e381680-a1a6-11e8-841f-8e8e7c315de8.png)
![screen shot 2018-08-16 at 10 44 26 pm](https://user-images.githubusercontent.com/39416177/44245298-1001da00-a1a6-11e8-82c6-b0a831d33628.png)
![screen shot 2018-08-16 at 10 44 33 pm](https://user-images.githubusercontent.com/39416177/44245299-11330700-a1a6-11e8-96ed-eea50fa932de.png)
![screen shot 2018-08-16 at 10 44 39 pm](https://user-images.githubusercontent.com/39416177/44245303-12fcca80-a1a6-11e8-8576-de4d308db09e.png)
![screen shot 2018-08-16 at 10 44 44 pm](https://user-images.githubusercontent.com/39416177/44245304-142df780-a1a6-11e8-870e-3f19a635a115.png)
![screen shot 2018-08-16 at 10 44 52 pm](https://user-images.githubusercontent.com/39416177/44245305-15f7bb00-a1a6-11e8-874e-9efd161dd19c.png)
![screen shot 2018-08-16 at 10 44 58 pm](https://user-images.githubusercontent.com/39416177/44245308-1728e800-a1a6-11e8-942e-f05e67a2fbd0.png)


## Details

**Headline:** 
The Deadliest Himalayan Mountains
**Published website version:**
https://jessimckenzi.github.io/himalayas/
**Code repository:**
https://github.com/jessimckenzi/data-studio/tree/master/03-expeditions/
**Final data set(s):** 
Himalayan Database
## What did you find to be the most difficult part of this project?
Narrowing down my focus and then going deep instead of broad.
## Are you satisfied with what you produced? Is there anything you would like to change or improve?
Meh—it's an unfortunate topic really because others have already done it better. I wish I had pursued the oxygen/no oxygen line of thought.
## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
